src/core/Microsoft.PowerToys.Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -97,6 +100,10 @@ public GeneralViewModel()
             _autoDownloadUpdates = GeneralSettingsConfigs.AutoDownloadUpdates;
             _isElevated = ShellPage.IsElevated;
             _runElevated = GeneralSettingsConfigs.RunElevated;
+
+            RunningAsUserDefaultText = loader.GetString("GeneralSettings_RunningAsUserText");
```

Comment:
@crutkas  I followed Ryan's suggestions of initializing these as const strings in the ViewModel (see line 32, 33), adding these strings to a resx would be redundant as they are already added in the strings resource file. Is there anything I am missing?
![image](https://user-images.githubusercontent.com/58791731/81984005-0f031400-95e9-11ea-98bd-5be8f0a8879a.png)